runtime/android/core_internal/src/org/xwalk/core/internal/XWalkHttpAuthHandlerInternal.java

```diff
@@ -12,11 +12,13 @@
  * @hide
  */
 @JNINamespace("xwalk")
-public class XWalkHttpAuthHandler {
+@XWalkAPI(createExternally = true)
+public class XWalkHttpAuthHandlerInternal {
 
     private long mNativeXWalkHttpAuthHandler;
     private final boolean mFirstAttempt;
 
+    @XWalkAPI
```

Comment:
If ```onReceivedHttpAuthRequest()```  was overried from customer's app, these APIs maybe called with ```XWalkHttpAuthHandler.proceed()/cancel()```, so I make ```proceed()``` and ```cancel()``` as XWalkAPI.